main.go

```diff
@@ -202,12 +205,23 @@ func getCommands() []cli.Command {
 				return cisetupcommand.RunCiSetupCmd()
 			},
 		},
+		{
+			Name:         "invite",
+			Usage:        invite.GetDescription(),
+			HelpName:     corecommon.CreateUsage("invite", invite.GetDescription(), invite.Usage),
+			ArgsUsage:    common.CreateEnvVars(),
+			BashComplete: corecommon.CreateBashCompletionFunc(),
+			Category:     otherCategory,
+			Action: func(c *cli.Context) error {
+				return invitecommand.RunInviteCmd(c)
+			},
+		},
```

Comment:
Let's comment out the above command, so that it doesn't become available for now.